openedx/core/djangoapps/content/course_overviews/migrations/0021_courseoverview_backfill_start_end_dates.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def copy_column_values(apps, schema_editor):
+    """
+    Copy start,end into start_date & end_date respectively
+    """
+    CourseOverview = apps.get_model('course_overviews', 'CourseOverview')
+    HistoricalCourseOverview = apps.get_model('course_overviews', 'HistoricalCourseOverview')
+
+    # Do not create historical rows for these changes, we will update historical records directly
+    CourseOverview.skip_history_when_saving = True
+
+    # Currently there are ~14k of these
+    for course_overview in CourseOverview.objects.all():
+        course_overview.start_date = course_overview.start
+        course_overview.end_date = course_overview.end
+        course_overview.save()
+
+    # Currently there are ~505k of these
```

Comment:
^ that being said, does it make sense to do this in a staggered/chunked fashion? We are using the cutoff for using gh-ost as ~1million

Reply:
I was going along with the work previously done on this, and figured that 1/2 of the max should be ok, but I'm open to doing this part as a mgmt command or the like if people are worried about it.